data/rulesets.ts

```diff
@@ -260,12 +260,11 @@ export const Rulesets: {[k: string]: FormatData} = {
 				for (const moveId of set.moves) {
 					const move = this.dex.moves.get(moveId);
 					const moveid = move.id;
-					if (hasMove[moveid]) continue;
+					if (hasMove[moveid]) return [species.baseSpecies + " has multiple copies of " + move.name + "."];
 					hasMove[moveid] = true;
 					moves.push(moveId);
```

Comment:
What's the point of having a `moves` variable? You're already iterating over `set.moves`, and you're no longer reassigning `set.moves`.

Reply:
Another good catch!